src/utils/validateUserSlippageTolerance.ts

```diff
@@ -0,0 +1,21 @@
+import { Percent } from '@uniswap/sdk-core'
+
+export enum SlippageValidationResult {
```

Comment:
These are enums, so you don't need to give them values - you can just let the compiler default-assign them.